packages/es-components/src/components/patterns/statusTracker/StatusTracker.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import genName from '../../util/generateAlphaName';
+
+const List = styled.ol`
+  margin: 0;
+  padding: 0;
+  position: relative;
+  z-index: 1;
+`;
+
+const ListItem = styled.li`
+  color: ${props => props.theme.colors.gray6};
+  float: left;
```

Comment:
Was flex considered over float, here?

Reply:
You know, I thought I had tried it (I hate floats) but perhaps I did not. Seems to work fine with flex, so I'll change it!

Reply:
Sweet!